source/Various/File2.lagda

```diff
@@ -0,0 +1,119 @@
+Alice Laroche, 26th September 2023
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K --exact-split #-}
+
+open import MLTT.Spartan
+open import UF.Univalence
+
+module Various.File2
+        (ua : Univalence)
+        (𝓤 : Universe)
+       where
+
+open import MLTT.NaturalNumbers
+open import UF.Base
+open import UF.DiscreteAndSeparated
+open import UF.Embeddings
+open import UF.Sets
+open import UF.Subsingletons
+open import W.Type
+open import Iterative.Multisets 𝓤
+open import Iterative.Multisets-Addendum ua 𝓤
+open import Iterative.Sets ua 𝓤
+open import Iterative.Sets-Addendum ua 𝓤
+open import Iterative.Ordinals ua 𝓤
+open import Various.File1 ua 𝓤
+
+φ≠ssupφ : (m : 𝕄) (x : 𝕄-root m) → 𝕄-forest m x ≠ m
```

Comment:
Two comments here. (1) We should think of a better name for this definition (I thought a bit but haven't come up with a good idea yet).

(2) Please use M,N to range over 𝕄, and A,B,C to range over 𝕍, and α, β, γ  to range over 𝕆, for the sake of consistency with the other files and the published literature.

Reply:
I replaced `φ≠ssupφ` by `𝕄-dont-contain-themselves : (M : 𝕄) → ¬ (M ⁅ M)` which imply it and is more intuitive, I think.